src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -340,7 +340,7 @@ private void move(Vector spdVec, double rot, boolean rateLimit) {
     var swerveModuleStates =
         DriveConstants.kDriveKinematics.toSwerveModuleStates(
             ChassisSpeeds.fromFieldRelativeSpeeds(
-                spdVec.x(), spdVec.y(), rotDelivered, Rotation2d.fromDegrees(-m_gyro.getAngle())));
+                spdCommanded.x(), spdCommanded.y(), rotDelivered, Rotation2d.fromDegrees(-m_gyro.getAngle())));
```

Comment:
According to Joseph this was not the only bug. The multiplication should happen after slew rate limitation, not before to mimic the old code.